packages/docs/operations-manual/monitoring.md

```diff
@@ -0,0 +1,177 @@
+
+- `consensus_istanbul_blocks_totalsigs`: The number of validators whose signatures were included in the child's parent seal. This can be used to determine how many validators are up and contributing to consensus. If this number falls towards two thirds of validator set size, network block production is at risk.
+
+- `consensus_istanbul_blocks_missedrounds`: Sum of the `round` included in the `parentAggregatedSeal` for the blocks seen. That is, the cumulative number of consensus round changes these blocks needed to make to get to this agreed block. This metric is only incremented when a block is succesfully produced after consensus rounds fails, indicating down validators or network issues.
+
+- `consensus_istanbul_blocks_validators`: (_Future release_) Total number of validators eligible to sign blocks.
+
+- `consensus_istanbul_core_consensus_count`: Count and timer for succesful completions of consensus (Use `quantile` tag to find percentiles: `0.5`, `0.75`, `0.95`, `0.99`, `0.999`)
+
+### Management APIs
+
+Celo blockchain inherits and extends go-ethereum's Javascript console, exposing [management APIs](https://geth.ethereum.org/docs/rpc/server) and [web3 DApp APIs](https://github.com/ethereum/wiki/wiki/JavaScript-API).
+
+Connect a client using a variant of the `attach` command line option:
+
+```bash
+geth attach --datadir DATADIR
+geth attach ipc:PATH/TO/geth.ipc
+geth attach http://localhost:8545
+geth attach ws://localhost:8546
+```
+
+<!--
+Celo adds specific functions around consensus:
+
+```bash
+
+```
+-->
+
+## Monitoring Attestation Service
+
+### Logging
+
+The Attestation Service provides JSON-format structured logs.
+
+### Metrics
+
+It also exposes the following Prometheus format metrics at `/metrics`:
+
+- `attestation_requests_total`: Counter for the number of attestation requests.
+
+- `attestation_requests_already_sent`: Counter for the number of attestation requests that were received but dropped because the local database records that they have already been completed.
+
+- `attestationRequestsWrongIssuer`: Counter for the number of attestation requests that were received but dropped because they specified the incorrect validator.
+
+- `attestationRequestsWOIncompleteAttestation`: Counter for the number of attestation requests that were received but when querying the blockchain no matching incomplete attestation could be found.
+
+- `attestationRequestsValid`: Counter for the number of requests received that are for the correct issuer and an incomplete attestation exists.
+
+- `attestationRequestsAttestationErrors`: Counter for the number of requests for which producing the attestation failed. This could be due to phone number or salt that does not match the hash, or the attestation was recorded fewer than 4 blocks ago.
+
+- `attestationRequestsUnableToServe`: Counter for the number of requests that could not be served because no SMS provider was configured for the phone number in the request.
```

Comment:
```suggestion
- `attestationRequestsUnableToServe`: Counter for the number of attestation requests that could not be served because no SMS provider was configured for the phone number in the request.
```